registry/recipes/packages/py-prompthandler/base.py

```
@@ -0,0 +1,128 @@
+
+        Raises:
+            ValueError: If the prompt with the given name is not found.
+            ValueError: If any input parameters are missing and the maximum recursion depth is reached.
+            ValueError: If any other error occurs while retrieving the prompt.
+
+        Returns:
+            str: The prompt content.
+        """
+        name = name.strip()
+        try:
+            content = self._get_prompt(name=name, *args, **kwargs)  # type: ignore[misc]
```

Ruff (0.14.14)

[warning] Star-arg unpacking after a keyword argument is strongly discouraged

(B026)